charts/dafka-consumer/templates/deployment.yaml

```diff
@@ -151,11 +151,11 @@ spec:
             {{ end }}
             {{ if .Values.retryPolicyExponentialBackoff}}
             - name: RETRY_POLICY_EXPONENTIAL_BACKOFF
-              value: {{ .Values.retryPolicyExponentialBackoff }}
+              value: {{ .Values.retryPolicyExponentialBackoff | squote }}
```

Comment:
This one is already a. list of comma separated numbers.
meaning no need to quote them...right?